coduno.go

```diff
@@ -56,10 +57,20 @@ func main() {
 	r.HandleFunc("/api/token/check/{token}", setupHandlerWithSessionStore(controllers.CheckToken))
 	r.HandleFunc("/api/mock", mockData)
 	r.HandleFunc("/api/mockCompany", mockCompany)
+	r.HandleFunc("/api/engineurl", setupHandler(getEngineURL))
 	r.HandleFunc("/subscriptions", setupHandler(mail.Subscriptions))
 	http.Handle("/", r)
 	appengine.Main()
 }
+
+func getEngineURL(w http.ResponseWriter, req *http.Request, ctx context.Context) {
+	if appengine.IsDevAppServer() {
+		w.Write([]byte("http://localhost:8081"))
+	} else {
+		w.Write([]byte("http://www.cod.uno/engine"))
```

Comment:
Please, do **never ever** refer to our public endpoints with HTTP (no S here) URLs. I know there is redirection in place that should work, but that
 1. introduces an additional redirect (and probably more than one roundtrip).
 2. Even worse, if the client does not correctly process HSTS, it will probably continue sending requests over HTTP, resulting in a redirect on *every request*.
 3. Possibly leaks private and/or secret data.